### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Philipp66904**
Tested in SITL

### CODE REVIEW

**Review by J0las — DISMISSED** [review]

File: launch/launch.py [hunk 1]
```diff
@@ -53,36 +70,37 @@ def generate_launch_description():
...
             package='qrcode_detection_package',
             executable='qr_code_scanner_node',
-            namespace=namespace,  
+            namespace=namespace,
             output='log',  # Ensure logging to file
             parameters=[{'sim': sim, 'IMG_PATH': img_directory}],
-            remappings=[('/rosout', [namespace, '/rosout'])],
-            arguments=['--prefix', 'taskset', '0x1000', '--ros-args', '--log-level', 'DEBUG']
+            remappings=[('/rosout', 'rosout')],
+            arguments=['--prefix', 'taskset', '0x1000',
```
Comment (J0las) [inline launch/launch.py]: Should the taskset stuff still be in the there?

Comment (Philipp66904) [reply launch/launch.py]: No, I will remove it

**Review by J0hn-Lawrence — APPROVED** [review]
Approved

**Comment by Philipp66904** [comment]
@J0las Please review

**Review by J0las — APPROVED** [review]

**Pull request merged**